pkg/cli/subcommands/workloadfilterlist/verifyCELConfig_nocel.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
nit: We should follow the Snake Case naming convention that is used in the rest of the repository.